runtime/aardvarks/info.yaml

```diff
@@ -0,0 +1,79 @@
+apiVersion: apigeeregistry/v1
+items:
+  - apiVersion: apigeeregistry/v1
+    kind: API
+    metadata:
+      name: apigee-apihub-demo-fauna-aardvarks-proxy
```

Comment:
I think we avoid encoding information into the name. All the information is available in the object, so let's leave the name the name... "fauna-aardvarks".